src/lib/models/keyframe.js

```diff
@@ -6,24 +6,30 @@ const validateSchema = require('jsonschema').validate
 const readFileAsync  = Promise.promisify(require('fs').readFile)
 
 const utils     = require('../shared/utils')
-const git       = require('../shared/git')
-const Component = require('../models/component')
-const Frame     = require('../models/frame')
 
 module.exports = class Keyframe {
-  static fromFile(path) {
-    return readFileAsync(path, 'utf8')
-    .then(this.fromYaml)
-  }
+  static fromFiles(paths) {
+    return Promise.map(paths, path => {
+      return readFileAsync(path, 'utf8').then(utils.parseYaml)
+    })
+    .then((kLayers) => {
+
+      // create and validate core keyframe.
+      var kf = kLayers[0]
+      new Keyframe(kf, true)
 
-  static fromYaml(yaml) {
-    return new Keyframe(utils.parseYaml(yaml))
+      // merge all layers(core keyframe + customizations on top)
+      for (let l of kLayers.slice(1)){
+        kf = _.merge(kf, l)
+      }
+      return new Keyframe(kf)
+    })
   }
 
-  constructor(obj, options = {}) {
-    const valid = validateSchema(obj, this.schema())
+  constructor(obj, isCore=false, options = {}) {
```

Comment:
I'd rather put the `isCore` in the options object. That way it avoids modifying the method signature, and since it's technically optional with a default, I think that makes more sense.